tools/sklearn/stacking_ensembles.py

```diff
@@ -82,7 +82,9 @@ def main(inputs_path, output_obj, base_paths=None, meta_path=None,
 
     weights = options.pop('weights', None)
     if weights:
-        options['weights'] = ast.literal_eval(weights)
+        weights = ast.literal_eval(weights)
+        if weights:
+            options['weights'] = weights
 
```

Comment:
do you also want to bump the version number in the XML file so the tool is updated?

Reply:
This is a simple bug fix. Is backporting possible?

Reply:
It is, but requires a manual update of the tool on usegalaxy.eu by @bgruening. I usually find it simpler to bump the version number also for small changes.